app/views/ThreadMessagesView/Dropdown/index.tsx

```diff
@@ -14,17 +13,19 @@ import DropdownItemHeader from './DropdownItemHeader';
 
 const ANIMATION_DURATION = 200;
 
-class Dropdown extends React.Component {
-	static propTypes = {
-		isMasterDetail: PropTypes.bool,
-		theme: PropTypes.string,
-		insets: PropTypes.object,
-		currentFilter: PropTypes.string,
-		onClose: PropTypes.func,
-		onFilterSelected: PropTypes.func
-	};
+interface IDropdownProps {
+	isMasterDetail: boolean;
+	theme: string;
+	insets: EdgeInsets;
+	currentFilter: string;
```

Comment:
You could type the way you did here to https://github.com/RocketChat/Rocket.Chat.ReactNative/pull/3538/files#diff-0522989489ae1ca9184bd55e549861fc816ac4e054faf5210ab3ad7bfd60feaaR93